packages/@repo/shared-modules.bundle/package.bundle.ts

```diff
@@ -0,0 +1,20 @@
+import {defineConfig, mergeConfig} from 'vite'
+
+import {defaultConfig} from '../package.bundle/src/package.bundle'
+
+export default defineConfig(() => {
+  return mergeConfig(defaultConfig, {
+    build: {
+      lib: {
+        entry: {
+          'react': './node_modules/react/cjs/react.production.min.js',
+          'react-dom': './node_modules/react-dom/cjs/react-dom.production.min.js',
+          'react-dom_server':
+            './node_modules/react-dom/cjs/react-dom-server-legacy.browser.production.min.js',
+          'react_jsx-runtime': './node_modules/react/cjs/react-jsx-runtime.production.min.js',
```

Comment:
I am converting anything in subpath of a package with `_` delimiter. Not really sure if this is needed since we can map an object key to a specific subpath and it can have benefits for dynamic routing in import maps. Thoughts?